--- C:/Users/peregarrett/Downloads/scenario_alt/scenario_alt/alt_day3.rpy	Tue Mar 03 13:05:50 2015
+++ C:/Users/peregarrett/Documents/GitHub/uvao/alt_day3.rpy	Thu Mar 05 11:34:27 2015
@@ -57,10 +57,12 @@
     $ alt_day3_sl_date = False #Дейтим Славю, два танца, помощь, вуайр в бане.
     $ alt_alice_dance = False #Разводим Двачевскую на танец.
     $ alt_day3_sl_dance = False #Затычка для Сыча на втором танце
     $ alt_day3_sl_dance2 = False
     
+	$ alt_day3_uvao_spotted = False #Заметили Юленьку у столовой
+	
 label alt_day3_start:
     $ alt_chapter(3, u"Утро")
     $ persistent.sprite_time = "day"
     $ day_time()
 

@@ -9201,11 +9203,11 @@
                     
                 label max_lp_select:
                     $ girls_list = ('dv', 'un', 'mi', 'sl')
                     $ lp_max = max( [eval('lp_'+w) for w in girls_list] )
                     $ max_who = [w for w in girls_list if eval('lp_'+w) == lp_max]
-                    $ max_who_id = max_who[0] if lp_max > 0 and len(max_who)==1 else 'nobody'
+                    $ max_who_id = max_who[0] if lp_max > 5 and len(max_who)==1 else 'nobody'
                     jump expression 'max_lp_{}'.format(max_who_id)
 
                 label max_lp_dv:
                     "Я смерил его взглядом."
                     me "Чудной ты какой-то. Иди уже."

@@ -9933,10 +9935,25 @@
                 "То, что называется «плавно слился», да?"
                 "Она подошла ко мне и, взяв за плечо, развернула в сторону танцпола."
                 mt "Пошли… Семён."
                 mt "Ты же не откажешь девушке в такой мелочи как танец?"
                 show mt smile dress close with dissolve
+                if alt_day1_genda_investigation and alt_day2_date != 132 and lp_un <= 5 and lp_sl <= 5 and lp_dv <= 5 and lp_mi <= 5 : # Не окучивали девушек, особенно Лену, даже через френдзон!
+                    menu:
+                        "Сбежать!":
+                            "Мысли тревожно заметались. {w} Танцевать не хотелось категорически, да и сосредоточиться на плане дальнейших действий танец бы только помешал, особенно вкупе со странным поведением вожатой." 
+                            "Решение, как всегда, нашлось своеобразное и эффективное."
+                            me "Ольга Дмитриевна, что-то живот у меня…"
+                            "Для достоверности я скрючился и взялся за якобы страдающий живот обеими руками."
+                            show mt normal dress with dissolve
+                            "Лицо моей несостоявшейся партнерши по танцам выражало сложную смесь недоверия, разочарования и беспокойства."
+                            mt "Беги уж, страдалец. Может, Виолу попросить потом в медпункт вернуться?"
+                            "Я энергично помотал головой и полетел в сторону туалета"
+                            hide mt with dissolve
+                            jump alt_day3_meet_uvao
+                        "Остаться.":
+                            pass
                 mt "Особеннно после всего случившегося."
                 "ЧЕГО СЛУЧИВШЕГОСЯ?"
                 me "Напоминаю, что в домик к себе подселили меня вы."
                 me "А то, что с утра я с кровати сверзился - я в этом не виноват."
                 if herc or loki:

@@ -11129,18 +11146,72 @@
        "И вообще, у меня Лена есть."
     elif alt_day3_mi_date:
         "И вообще, у меня Мику есть."
 
     "Помывшись, я уступил помещение Мику и Лене, подумал было о том, что… И чуть ли не силком уволок себя от бревенчатого здания. Надо спать. Спаааать!"
+    jump alt_day3_sleeptime
+
+label alt_day3_meet_uvao:
+    scene bg ext_dining_hall_away_night 
+    with dissolve
+    play ambience ambience_camp_center_night fadein 1
+    th "Фух, оторвался."
+    dreamgirl "Ну ты даешь! Такую даму бортануть!{w} Ты что, ослеп и оглох, упускать подобное? Поиметь вожатую!{w} Все, знать тебя не знаю!"
+    th "Оставь меня, старушка, я в раздумьях.{w} Тебе, может быть, и по барабану, где я оказался, но мне-то явно нет. И где, и почему, и когда…"
+    dreamgirl "Так точно, мон женераль, все силы на раскрытие заговора по перемещению Семенов в пространстве и времени!"
+    "Под этот разговор я сел прямо на траву и неспешно озирал окрестности, собирая мысли в кучку."
+    "Исчезающие на глазах тени, странности в поведении местных обитательниц…"
+    "Додумать я эту мысль не успел, потому как мое внимание привлек темный силуэт, залезающий в окно столовой."
+    scene ext_dining_hall_away_night_uvao2 with dissolve
+    play music music_list["mystery_girl_v2"] fadein 3
+    th "Алиса!"
+    "пришло в голову очевидное объяснение."
+    th "И не первый раз уже!"
+    dreamgirl "Может ее того… припугнуть немного?"
+    "Идея припугнуть ДваЧе была достаточно привлекательной, чтобы я поднялся с насиженного места и начал тихо подбираться к приоткрытому окну."
+    scene ext_dining_hall_away_night_uvao2:
+        xalign 0.5 yalign 0.5 zoom 1.0
+        linear 1.0 zoom 2.0 xalign 0.7 yalign 0.5
+    
+    "Примерно на половине пути окно опять распахнулось и тень начала выбираться."
+    "Я только вознамерился окликнуть Алису, как заметил явное несоответствие по ряду пунктов."
+    "Во-первых, девушка была явно пониже и пофигуристей, {w} во-вторых, одета она была не в пионерскую форму,{w} а в-третьих…"
+    "Для осознания этого «в-третьих» я глубоко вдохнул и присмотрелся еще раз. Да, у нее действительно были кошачьи уши и аккуратный хвост!"
+    dreamgirl "Вау, значит, такой маскарад и при совке был популярен!"
+    scene ext_dining_hall_away_night_uvao1 with dissolve:
+        zoom 2.0 xalign 0.7 yalign 0.5
+    "Силуэт дернул ухом и скрылся в тенях за углом столовой. Я остолбенел."
+    scene bg ext_dining_hall_away_night with dissolve:
+        zoom 2.0 xalign 0.7 yalign 0.5
+        linear 1.0 zoom 1.0 xalign 0.5 yalign 0.5
+    dreamgirl "Вот они, минусы юности! От гормонов уже глюки начинаются."
+    th "И окно от ветра открывалось? Нет, кто-то тут явно был."
+    dreamgirl "Что же ты ее на «кис-кис-кис» не подозвал?"
+    th "Ха-ха, как смешно."
+    dreamgirl "А если серьезно – это уже диагноз, друг ситный. {w} Надо было тебе меньше аниме смотреть и хентайной манги почитывать."
+    th "Но я же полностью нормальный…"
+    dreamgirl "…и справка есть, знаю.{w} А еще ты разговариваешь с голосом у себя в голове и находишься в пионерлагере. Нафиг-нафиг."
+    "Я оторвал остановившийся взгляд от стены столовой и побрел к себе в домик. Увиденное требовало осмысления, причем осмысления по возможности свежей головой."
+    $ alt_day3_uvao_spotted = True
+    stop music fadeout 3
+    jump alt_day3_sleeptime
     
 label alt_day3_sleeptime:
 
     $ night_time ()
     $ persistent.sprite_time = "night"
     play ambience ambience_camp_center_night fadein 1
     play music music_list["a_promise_from_distant_days_v2"] fadein 2
-    if alt_day3_dv_date:
+    if alt_day3_uvao_spotted:
+        scene bg ext_house_of_mt_night_without_light with fade
+        "Домик встретил меня тишиной и покоем. Ольги Дмитриевны не было ни внутри, ни на подходе."
+        pause(1)
+        stop ambience fadeout 1
+        scene bg int_house_of_mt_night2 with fade
+        "Я прилег на кровать и попробовал еще раз объединить все увиденное и понятое в более-менее цельную картинку."
+        "Картинка не складывалась, но чувствовалось, что из всего происходящего торчат чьи-то кошачьи уши."
+    elif alt_day3_dv_date:
         window hide
         scene bg ext_house_of_el_night with dissolve
         pause(1)
         play sound sfx_open_dooor_campus_1
         scene int_extra_house with dissolve
@@ -11266,14 +11337,14 @@
        "И глупая, и смешная ситуация в костюмерной."
     if alt_day3_mi_dance0 and not alt_miku_dj:
         "Уроки танцев…"
     if alt_day3_mi_dance2:
         "…Вечерний танец"
-    if alt_day3_mi_invite:
-        extend ", мы оба в великолепных нарядах, и невольная гордость за то, что она - такая красивая."
-    else:
-        extend " куда более горячий."
+        if alt_day3_mi_invite:
+            extend ", мы оба в великолепных нарядах, и невольная гордость за то, что она - такая красивая."
+        else:
+            extend ", куда более горячий."
     if alt_day3_mi_invite and not alt_miku_dj:
         "Дурацкая сценка с вожатой, которая подумала ровно то, что должна была подумать."
     if alt_miku_dj:
         "Японская девочка в огромных наушниках за вертушками."
     if alt_day3_peep_show:
@@ -11293,10 +11364,12 @@
         "Вечер наперевес с гитарой."
     if alt_day3_bugs == 1:
         "Жуки, которых мы с Ульянкой наловили и подложили злюке-вожатой."
     if alt_day3_peep_show:
         "… Сценка в бане…"
+    if alt_day3_uvao_spotted:
+        "Странное существо возле столовой…"
     "… Денёк вышел тот ещё…"
     "С последней мыслью я уснул."
     stop music
     stop ambience
     stop sound_loop
@@ -11365,10 +11438,18 @@
             $ routetag = "sl"
             jump alt_day4_start_sl
         elif alt_day3_sl_invite_beach and alt_day3_sl_dance and alt_day3_sl_dance2:
             $ routetag = "sl7dl"
             jump alt_day4_start_sl_7dl
+    else:
+        jump alt_day3_router_neutral
+    
+label alt_day3_router_neutral:
+    if alt_day3_uvao_spotted:
+        $ routetag = "uv"
+        "Сон был тревожным…"
+        jump alt_day4_start_uvao #Начало ЮВАО-рута
     else:
         "И спалось мне не очень."
         window hide
         $ routetag = "us"
         jump alt_day4_start_neutral
